src/app/demo/templates/templates.component.html

```diff
@@ -114,6 +114,9 @@ <h2 ngNonBindable>Event binding: (event)</h2>
       [readonly]="true"
       type="html">
     </slides-code-editor>
+    <div class="info">
+      Parentheses <b>(click)</b> are used for event binding, but "on-" can also be used <b> on-click</b>
```

Comment:
It's indeed true, also square braces are equivalent to bind, docs are here: 
https://angular.io/docs/ts/latest/guide/template-syntax.html#!#binding-syntax (search for `on-` or `bind-`)